pkg/operator/controller/ingress/status.go

```diff
@@ -822,25 +819,28 @@ func computeDNSStatus(ic *operatorv1.IngressController, wildcardRecord *iov1.DNS
 			},
 		}
 	}
-
-	if !manageDNSForDomain(ic.Status.Domain, status, dnsConfig) {
-		return []operatorv1.OperatorCondition{
-			{
-				Type:    operatorv1.DNSManagedIngressConditionType,
-				Status:  operatorv1.ConditionFalse,
-				Reason:  "DomainNotMatching",
-				Message: "DNS management is not supported for ingresscontrollers with domain not matching the baseDomain of the cluster DNS config.",
-			},
-		}
-	}
-
-	conditions := []operatorv1.OperatorCondition{
-		{
+	var conditions []operatorv1.OperatorCondition
+	if ic.Status.EndpointPublishingStrategy.LoadBalancer.DNSManagementPolicy == operatorv1.UnmanagedLoadBalancerDNS {
+		conditions = append(conditions, operatorv1.OperatorCondition{
+			Type:    operatorv1.DNSManagedIngressConditionType,
+			Status:  operatorv1.ConditionFalse,
+			Reason:  "UnmanagedLoadBalancerDNS",
+			Message: "The DNS management policy is set to Unmanaged.",
+		})
+	} else if !manageDNSForDomain(ic.Status.Domain, status, dnsConfig) {
```

Comment:
Same question as for `desiredWildcardDNSRecord`: Is it necessary to check `domainMatchesBaseDomain` here if `Reconcile` or `setDefaultPublishingStrategy` sets `ic.Status.EndpointPublishingStrategy.LoadBalancer.DNSManagementPolicy`?